gc-worker/server/server_test.go

```diff
@@ -108,3 +158,62 @@ func NewTestSingleConfig() *embed.Config {
 	cfg.ClusterState = embed.ClusterStateFlagNew
 	return cfg
 }
+
+func TestCalcNewGCSafePoint(t *testing.T) {
+	cfg := NewConfig()
+	ctx := context.Background()
+	s := &Server{
+		cfg:            cfg,
+		ctx:            ctx,
+		startTimestamp: time.Now().Unix(),
+	}
+	defer s.Close()
+	newSp := s.calcNewGCSafePoint(0, 100)
+	require.Equal(t, newSp, uint64(100))
+
+	newSp = s.calcNewGCSafePoint(1000, 100)
+	require.Equal(t, newSp, uint64(100))
+
+	newSp = s.calcNewGCSafePoint(1000, 10000)
+	require.Equal(t, newSp, uint64(1000))
+
+}
+
+func TestCalcGcSafePoint(t *testing.T) {
```

Comment:
Suggest to make naming be consistent.

```suggestion
func TestCalcGCSafePoint(t *testing.T) {
```

Reply:
done